daemon/cmd/status.go

```diff
@@ -165,15 +165,15 @@ func (d *Daemon) getIPV4BigTCPStatus() *models.IPV4BigTCP {
 
 func (d *Daemon) getBandwidthManagerStatus() *models.BandwidthManager {
 	s := &models.BandwidthManager{
-		Enabled: option.Config.EnableBandwidthManager,
+		Enabled: d.bwManager.Enabled(),
 	}
 
-	if !option.Config.EnableBandwidthManager {
+	if !d.bwManager.Enabled() {
 		return s
 	}
 
 	s.CongestionControl = models.BandwidthManagerCongestionControlCubic
-	if option.Config.EnableBBR {
+	if d.bwManager.BBREnabled() {
```

Comment:
Are we moving all `option.Config.*` to their respective modules? Can you share more context for these changes?

Reply:
We do not have hard rules about this, in general yes. However, we do not yet have solutions for all parts of `option.Config`. The situation currently is that it contains both settings and runtime config/information which we want to separate from each other. 

In this case there are reasons why bandwidth manager can't be enabled even if it has been requested to. Instead of modifying the `EnableBandwidthManager` config to reflect this we provide the runtime info via these methods now. That way there is a clear distinction between requested state and actual state.